templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -180,8 +181,9 @@ contents:
       nmcli device disconnect $iface
 
       if ! nmcli connection show ovs-if-phys0 &> /dev/null; then
+        # use ${extra_phys_args[@]+"${extra_phys_args[@]}"} instead of ${extra_phys_args[@]} to be compatible with bash 4.2 in RHEL7.9
```

Comment:
can you please comment on why bash4.2 on rhel7.9 needs the complicated syntax to parse `extra_phys_args` array?

Reply:
prior bash 4.4, using `${extra_phys_args[@]}` when `extra_phys_args` is empty and the nounset option is enabled, it throws an unbound variable error. 

https://git.savannah.gnu.org/cgit/bash.git/tree/CHANGES?id=3ba697465bc74fab513a26dea700cc82e9f4724e#n878

Reply:
interesting never seen this pattern b4 to help with readability can u store the new form of extra_phys_args[@] in new var and use it below

Reply:
fixed.